test/e2e/run_test.go

```diff
@@ -1475,6 +1475,32 @@ VOLUME %s`, ALPINE, volPath, volPath)
 		Expect(numContainers).To(Equal(0))
 	})
 
+	It("podman run after infra-container rootfs removed", func() {
```

Comment:
please add a link to the issue here, it is always good to know what test covers which issue without having to get this info from git.